### ISSUE TITLE
Before first headline at position 94 in buffer config.org.vRvaJZ

### ISSUE THREAD

**Issue by ineu** [hlissner/doom-emacs#3688]
After `doom up` I've got the subject error. Tried removing the whole `~/.emacs.d` dir, then git cloned the repo, but see the same issue on `doom i`:

```
❯ doom i
Installing straight...
Cloning use-package...
Cloning use-package...done
Building use-package...
Building use-package → Building bind-key...
Building use-package → Building bind-key...done
Building use-package...
Building use-package...done
> Compiling your literate config...
Before first headline at position 94 in buffer config.org.vRvaJZ
```

None of my local configs from `~/.doom.d` were changed recently.
The previous `doom up` happened around two days ago.

**Comment by hlissner** [comment]
My bad, this was a regression from 8580b53e8. As of 01c5c35 this should be fixed. Let me know if that isn't the case and I'll reopen it. Thanks for bringing it to my attention!

**Comment by ineu** [comment]
Have another issue now:
```
❯ rm -rf .emacs.d
❯ git clone https://github.com/hlissner/doom-emacs ~/.emacs.d
Клонирование в «/home/ineu/.emacs.d»…
remote: Enumerating objects: 294, done.
remote: Counting objects: 100% (294/294), done.
remote: Compressing objects: 100% (162/162), done.
remote: Total 89155 (delta 149), reused 220 (delta 131), pack-reused 88861
Получение объектов: 100% (89155/89155), 21.77 MiB | 5.22 MiB/s, готово.
Определение изменений: 100% (62948/62948), готово.
❯ doom i
Installing straight...
Cloning use-package...
Cloning use-package...done
Building use-package...
Building use-package → Building bind-key...
Building use-package → Building bind-key...done
Building use-package...
Building use-package...done
> Compiling your literate config...
Wrong type argument: stringp, nil
```

**Comment by hlissner** [comment]
> Wrong type argument: stringp, nil

This should be fixed as of 00a8f750e.

**Comment by ineu** [comment]
Works for `doom i` and `doom up`. Thank you.
